model/classification.py

```diff
@@ -118,7 +118,8 @@ class Subject(Base):
     # the book's audience.
     audience = Column(
         Enum("Adult", "Young Adult", "Children", "Adults Only",
-             name="audience"),
+            "All Ages", "Research",
```

Comment:
This also threw an error when trying to save the new value in the admin.

Reply:
This makes sense.